src/app/main/component/comments/components/comment-textarea/comment-textarea.component.ts

```diff
@@ -267,6 +283,29 @@ export class CommentTextareaComponent implements OnInit, AfterViewInit, OnChange
     });
   }
 
+  private updateLinksInTextarea(currentText: string): void {
+    if (Patterns.urlLinkifyPattern.test(currentText)) {
+      this.commentTextarea.nativeElement.innerHTML = this.renderLinks(currentText);
+      this.initializeLinkClickListeners(this.commentTextarea.nativeElement);
+    }
+  }
+
+  renderLinks(text: string): string {
+    return text.replace(Patterns.urlLinkifyPattern, (match) => {
+      return `<a href="${match}" target="_blank" rel="noopener noreferrer">${match}</a>`;
+    });
+  }
+
+  initializeLinkClickListeners(element: HTMLElement): void {
+    const links = element.querySelectorAll('a');
+    links.forEach((link: HTMLAnchorElement) => {
```

Comment:
Selects all anchor tags in the element every time it's called. This might not be the most efficient